conf/local.conf.sample

```diff
@@ -6,9 +6,8 @@ DISTRO = 'sokol-flex'
 # qemuarm, qemumips, qemuppc, qemux86, qemux86-64.
 MACHINE ??= "qemux86"
 
-# The path to the installed sourcery toolchain. This defaults to using the one
-# from the CodeBench we're installed in.
-#EXTERNAL_TOOLCHAIN ?= "/path/to/toolchain"
+# The path to the toolchains installed with CodeBench.
+#TOOLCHAINS_PATH ?= "/path/to/toolchains"
```

Comment:
Would it make sense to clarify what the default would be in case it isn't set here?

Reply:
Yes, good idea, thanks. I'll update it.

Reply:
Updated, thanks!